src/components/molecules/FollowerGoal.tsx

```diff
@@ -1,7 +1,7 @@
 import { useTwitchFollowers } from '../../hooks/twitch';
 import { LoadingBarClassNames, LoadingBar } from './internal/LoadingBar';
 
-interface FollowerGoalProps {
+export interface FollowerGoalProps {
```

Comment:
Stories would cry if this is not exported

Reply:
It works fine here without exporting it, can you review those added exports?

Reply:
![image](https://user-images.githubusercontent.com/764128/101669494-e7264280-3a49-11eb-9d28-c373081612c2.png)
I get this issue from IntelliJ if I don't export it. I don't get this error with the `typecheck` command and everything works, so I am puzzled now.